src/main.ts

```diff
@@ -5,6 +5,7 @@ async function run(): Promise<void> {
   try {
     const githubToken = core.getInput('github_token', {required: true})
     const uploadUrl = core.getInput('upload_url', {required: true})
+    const releaseUrl = core.getInput('release_url', {required: true})
```

Comment:
We can get the base url of the GitHub API from the `GITHUB_API_URL` environment value.
https://docs.github.com/en/actions/reference/environment-variables

please use it instead of a new parameter.

Reply:
@shogo82148 
I changed from parameter to environment value.